datacatalog/cloud-client/src/main/java/com/example/datacatalog/CreateFilesetEntry.java

```diff
@@ -0,0 +1,137 @@
+      } catch (Exception e) {
+        System.out.println("Entry does not exist.");
+      }
+
+      // Delete any pre-existing Entry Group with the same name
+      // that will be used in step 2.
+      try {
+        dataCatalogClient.deleteEntryGroup(
+                EntryGroupName.of(projectId, location, entryGroupId).toString());
+      } catch (Exception e) {
+        System.out.println("Entry Group does not exist.");
+      }
+
+      // -------------------------------
+      // 2. Create an Entry Group.
+      // -------------------------------
+      // Construct the EntryGroup for the EntryGroup request.
+      EntryGroup entryGroup = EntryGroup.newBuilder().build();
+
+      // Construct the EntryGroup request to be sent by the client.
+      CreateEntryGroupRequest entryGroupRequest = CreateEntryGroupRequest.newBuilder()
+              .setParent(LocationName.of(projectId, location).toString())
+              .setEntryGroupId(entryGroupId)
+              .setEntryGroup(entryGroup)
+              .build();
+
+      // Use the client to send the API request.
+      EntryGroup entryGroupResponse = dataCatalogClient.createEntryGroup(entryGroupRequest);
+
+      System.out.printf("\nEntry Group created with name: %s\n", entryGroupResponse.getName());
+
+      // -------------------------------
+      // 3. Create a Fileset Entry.
+      // -------------------------------
+      // Construct the Entry for the Entry request.
+      Entry entry = Entry.newBuilder()
+              .setDisplayName("My Fileset")
+              .setDescription("This fileset consists of ....")
+              .setGcsFilesetSpec(GcsFilesetSpec.newBuilder().addFilePatterns("gs://my_bucket/*")
+                      .build())
+              .setSchema(Schema.newBuilder()
+                      .addColumns(ColumnSchema.newBuilder()
+                              .setColumn("first_column")
+                              .setType("STRING")
+                              .setDescription("This columns consists of ....").build())
+                      .addColumns(ColumnSchema.newBuilder()
+                              .setColumn("second_column")
+                              .setType("STRING")
+                              .setDescription("This columns consists of ....").build())
+                      .build())
+              .setType(EntryType.FILESET)
+              .build();
+
+      // Construct the Entry request to be sent by the client.
+      CreateEntryRequest entryRequest = CreateEntryRequest.newBuilder()
+              .setParent(entryGroupResponse.getName())
+              .setEntryId(entryId)
+              .setEntry(entry)
+              .build();
+
+      // Use the client to send the API request.
+      Entry entryResponse = dataCatalogClient.createEntry(entryRequest);
+
+      System.out.printf("\nEntry created with name: %s\n", entryResponse.getName());
+
+
+    } catch (Exception e) {
```

Comment:
This Exception is too broad as well.

Reply:
Changed, also now explaining what could cause the exceptions.